src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.animal.AnimalComparator;
+
+
+/**
+ * Sorts all animals in the zookeep book to the user.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sorts the animals according to their name, ID or earliest feedtime. \n"
+            + "Parameters: keyword for sorting category (name, id, feedtime) \n"
```

Comment:
Think this line can be standardised with the rest of the commands? I suggest something like
`Parameters: KEYWORD (name, id, feedtime)`? ok not sure how to phrase the stuff in the brackets but something like this would be nicer i think